test/integration/targets/aws_lambda/tasks/main.yml

```diff
@@ -134,16 +135,46 @@
            - 'result is not failed'
            - 'result.result.output.message == "hello Mr Ansible Tests"'
 
+    - name: test lambda config updates
+      lambda:
+        name: "{{lambda_function_name}}"
+        runtime: "nodejs10.x"
+        tracing_mode: 'Active'
+        handler: "mini_lambda.handler"
+        role: "ansible_lambda_role"
+      register: update_result
+
+    - name: assert that update succeeded
+      assert:
+        that:
+          - update_result is not failed
+          - update_result.changed == True
+          - update_result.configuration.runtime == 'nodejs10.x'
+          - update_result.configuration.tracing_config.mode == 'Active'
+
+    - name: reset config updates for the following tests
```

Comment:
It's still missing the changed == False test above this one. That's how I found the modification bug. Here's what I had:
```
    - name: test no changes are made with the same parameters
      lambda:
        name: "{{lambda_function_name}}"
        runtime: "nodejs10.x"
        tracing_mode: 'Active'
        handler: "mini_lambda.handler"
        role: "ansible_lambda_role"
      register: update_result

    - name: assert that update succeeded
      assert:
        that:
          - update_result is not failed
          - update_result.changed == False
          - update_result.configuration.runtime == 'nodejs10.x'
          - update_result.configuration.tracing_config.mode == 'Active'
```

Reply:
Also done now